doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,300 @@
+        set start time = 0
+        set end time   = 50
+      end
+    end
+
+---------------------------
+Running the DEM Simulation
+---------------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``lethe-particles`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+  :class: copy-button
+
+  lethe-particles dem-packing-in-spouted-bed.prm
+
+or in parallel (where 8 represents the number of processors)
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 8 lethe-particles dem-packing-in-spouted-bed.prm
+
+After the particles have been packed inside the square bed, we can move on to the fluid-particles simulation.
+
+
+-----------------------
+CFD-DEM Parameter File
+-----------------------
+
+The CFD simulation is to be carried out using the packed bed simulated in the previous step. Here we will focus on the modified section as well.
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation is run for 5 s with a time step of 0.001 s. The time scheme and setting for output is shown as follows.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method               = bdf1
+      set number mesh adapt    = 0
+      set output name          = result_
+      set output frequency     = 50
+      set startup time scaling = 0.6
+      set time end             = 5
+      set time step            = 0.001
+      set subdivision          = 1
+      set log precision        = 10
+      set output path          = ./output/
+    end
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Regarding the boundary conditions, we apply slip boundary condition to the wall, a uniform Dirichlet boundary condition at the bottom of the small channel, and free-stream to the top of the cylinder. The following schematic figure describes the ID of each boundary and the positon of the floating wall.
+
+.. image:: images/ID.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: ID
+    :height: 10cm
+
+we set the inlet velocity to 2.5 m/s, and we do not impose the background velocity on the bottom of the cylinder as in the previous spouted bed example. The value of beta on the outlet boundary was set to 100, which is relatively high, to stabilize the simulation.
+
+.. code-block:: text
+
+  subsection boundary conditions
+    set time dependent = false
+    set number         = 5
+
+
+    subsection bc 0 #outlet
+      set id   = 3
+      set type = outlet
+      set beta = 100
+    end
+
+    subsection bc 1 #inlet
+      set id   = 2
+      set type = function
+      subsection u
+        set Function expression = 0
+      end
+      subsection v
+        set Function expression = 0
+      end
+      subsection w
+        set Function expression = 2.5
+      end
+    end
+
+    subsection bc 2 #bed_wall
+      set id = 6
+      set type = slip
+    end
```

Comment:
What is bed wall?

Reply:
I wanted to express like this is the wall of main geometry, not small channel. 
Maybe I need to change this into just "wall"